### DISCUSSION TITLE
Hello, Github community. Greetings for the first time.

### DISCUSSION THREAD

**Discussion by YuHyeonWook** [community/community#139201] [Accessibility]
- Hello

**Comment by queenofcorgis** [comment]
👋 Welcome to the GitHub Community on Discussions, @YuHyeonWook  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).
